### DISCUSSION TITLE
2023/09/12/gitbook-v2-intro

### DISCUSSION THREAD

**Discussion by giscus** [kaochenlong/kaochenlong-discuss#11] [General]
# 2023/09/12/gitbook-v2-intro

《為你自己學 Git（第二版）》是一本深入介紹 Git 版本控制系統的實用指南。無論你是新手還是有經驗的 Git 使用者，這本書都提供了清晰的解釋和示例，幫助你理解 Git 的核心概念和常用指令。作者以淺顯易懂的方式解釋了 Git 的運作原理，並分享了在多種情境下如何使用 Git

https://kaochenlong.com/2023/09/12/gitbook-v2-intro.html

**Comment by dnxxymcj** [comment]
您好，很喜欢这本书，请问第二版什么时候能出版啊？

**Comment by kaochenlong** [comment]
希望可以是 2025 年的第一季 :)

**Reply by tammychang85** [reply]
請問大大今年有機會拜讀到第二版嗎Q

**Reply by kaochenlong** [reply]
今年剩 2 個月大概沒機會，明年一定！

**Reply by tammychang85** [reply]
期待期待!

**Reply by ivanke** [reply]
期待 +1
